### ISSUE TITLE
Information about compiling

### ISSUE THREAD

**Issue by orzel** [ghidraninja/game-and-watch-doom#3]
A raw clone will fail with
In file included from Core/Src/flash.c:1:
Core/Inc/flash.h:4:10: fatal error: stm32h7xx_hal.h: No such file or directory
    4 | #include "stm32h7xx_hal.h"
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.

Can you provide more information about compiling ? What's the preferred way of installing the sdk ? Especially (for me) in linux.

Should we go the way of a full blown CMSIS install ? Or kinda use STM32CubeMX ? Any other method ?

**Comment by benob** [comment]
You need to link/copy the Drivers directory from STM32Cube in the base directory of the project. You also need to copy startup_stm32h7b0xx.s in that directory. Check https://github.com/kbeckmann/game-and-watch-retro-go, you have to follow the same structure.

**Comment by orzel** [comment]
Indeed, it works a lot better with this (from the main directory of this repository clone):
```
ln -s ../game-and-watch-retro-go/Drivers/
ln -s ../game-and-watch-retro-go/startup_stm32h7b0xx.s
```

It now fails with 
```
/usr/libexec/gcc/arm-none-eabi/ld: build/gw_doom.elf section `.data' will not fit in region `FLASH'
/usr/libexec/gcc/arm-none-eabi/ld: region `FLASH' overflowed by 2336 bytes
collect2: error: ld returned 1 exit status
make: *** [Makefile:260: build/gw_doom.elf] Errore 1
```

Do I need to specifiy the flash size somewhere ? I happen to have 16MB instead of 1MB.
